src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+public class ColorSupplier extends Lottery {
+
     public String getRandomColor() {
-        return null;
+
+        int randomIndex = getRandom().nextInt(Color.values().length);
```

Comment:
Create object Random outside of method, as class field.
private Random random = new Random();